### PR TITLE
New approach envoy v3- part 4

### PR DESCRIPTION
**Pull request by arianaw66**

### CODE REVIEW

**Comment by arianaw66** [comment]
/kick again- gateway kube2e tests passed locally

File: projects/sds/pkg/server/server.go [hunk 1]
```diff
@@ -14,12 +14,12 @@ import (
 
 	"google.golang.org/grpc"
 
-	auth "github.com/envoyproxy/go-control-plane/envoy/api/v2/auth"
-	core "github.com/envoyproxy/go-control-plane/envoy/api/v2/core"
-	sds "github.com/envoyproxy/go-control-plane/envoy/service/discovery/v2"
+	core "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
```
Comment (EItanya) [inline projects/sds/pkg/server/server.go:17]: In my v2 -> v3 transition PR I fixed the SDS server to support v2 and v3 simultaneously. Can we pull that in?

**Comment by arianaw66** [comment]
Metrics, sds, and accesslogger all need to support both v2 and v3 simultaneously. Tabling this PR until we're on to that part of this issue.

**Comment by kdorosh** [comment]
closing as stale, v3 API work has been merged https://github.com/solo-io/gloo/pull/3894

**Pull request closed without merging**